docs/src/maintain/review-pull-requests.md

```diff
@@ -37,28 +37,52 @@ Once the bot checks have been satisfied, you check the following:
 
 **Note:** If you are a team member and you've left a comment on the pull request, please follow up to verify that your comments have been addressed.
 
-## Who Can Merge a Pull Request
+## Required Approvals for Pull Requests
 
-TSC members, Reviewers, Committers, and Website Team Members may merge pull requests, depending on the contents of the pull request.
+Any committer, reviewer, or TSC member may approve a pull request, but the approvals required for merging differ based on the type of pull request.
 
-Website Team Members may merge a pull request in the `eslint.org` repository if it is:
+One committer approval is required to merge a non-breaking change that is:
 
 1. A documentation change
+1. A bug fix (for either rules or core)
 1. A dependency upgrade
+1. Related to the build
 1. A chore
 
-Committers may merge a pull request if it is a non-breaking change and is:
+For a non-breaking feature, pull requests require approval from one reviewer or TSC member, plus one additional approval from any other team member.
+
+For a breaking change, pull requests require an approval from two TSC members.
+
+::: important
+If you approve a pull request and don't merge it, please leave a comment explaining why you didn't merge it. You might say something like, "LGTM. Waiting three days before merging." or "LGTM. Requires TSC member approval before merging" or "LGTM. Would like another review before merging."
+:::
+
+## Moving a Pull Request Through the Triage Board
+
+When a pull request is created, whether by a team member or an outside contributor, it is placed in the "Needs Triage" column of the Triage board automatically. The pull request should remain in that column until a team member begins reviewing it.
+
+If the pull request does not have a related issue, then it should be moved through the normal [triage process for issues](./manage-issues) to be marked as accepted. Once accepted, move the pull request to the "Implementing" column.
+
+If the pull request does have a related issue, then move the pull request to the "Implementing" column.
```

Comment:
Normally I moved to "Implementing" only if the corresponding issue has been accepted, otherwise I moved to "evaluating". This needs to change?

Reply:
That's a good point. @mdjermanovic @fasttime what do you think?

Reply:
I agree, "evaluating" makes more sense than "implementing" when the corresponding issue hasn't been accepted yet.